### ISSUE TITLE
Help beta-testing ThirdStats

### ISSUE THREAD

**Issue by devmount** [devmount/third-stats#370]
Hey there stats loving folks!

Due to several reasons ThirdStats got much less attention from me than I would have liked in the last months and in general I could use some help testing the ThirdStats nightly builds. So if you want to support this project and don't want to [donate money](https://paypal.me/devmount) or contribute code - testing and reporting bugs is a great way to help! Here's how it works:

<img src="https://user-images.githubusercontent.com/5441654/158936290-fd8e6709-78a9-40dd-9a5a-720bd1a40b44.png" width="40%" align="right" />

1. Download the [latest nightly build from the ThirdStats CDN](https://third-stats.cdn.devmount.com/) (XPI file)
2. Start Thunderbird and install the XPI file by going to _Settings_ > _Add-ons and Themes_ > _Extensions_ > _Gear_ Button > _Install Add-on from file_ > Choose the downloaded XPI file
3. Now start using ThirdStats and check if...
    - ThirdStats logo appears in the menu and the popup shows all active accounts
    - add-on options page displays properly, alle options can be changed, options can be reset
    - stats page display properly, data is shown correctly, processing works, all filter work as expected

If anything doesn't work, [file a bug report](https://github.com/devmount/third-stats/issues/new?template=bug_report.md) or [start a discussion](https://github.com/devmount/third-stats/discussions) about it.
Thanks a lot for your support 💚

**Comment by koobs** [comment]
@devmount Anyway we can get nightly builds to auto update? In the event that's not possiblem idea:

- In-extension version check, reports out of date / new available build in some manner (toolbar icon? notification?)

**Comment by devmount** [comment]
An auto update would indeed make the testing of the current build much easier. Unfortunately there is no such mechanism for no-ATN add-ons (at least not known to me). So if some day TB implements something like that, I will adapt it right away.

The in-extension version check is a great idea. But that would require making calls either to GitHub or the ThirdStats CDN which I would like to avoid for privacy reasons. Currently ThirdStats works without any external calls.

**Comment by koobs** [comment]
> An auto update would indeed make the testing of the current build much easier. Unfortunately there is no such mechanism for no-ATN add-ons (at least not known to me). So if some day TB implements something like that, I will adapt it right away.
> 
> 
> 
> The in-extension version check is a great idea. But that would require making calls either to GitHub or the ThirdStats CDN which I would like to avoid for privacy reasons. Currently ThirdStats works without any external calls.

Understood. How about

- [ ] Check and notify me of updated development versions

Opt-in only :)

**Comment by devmount** [comment]
Yes, opt-in would be a possibility. If I receive feedback from some more people who are using/testing nightly builds, I'll implement it.
